library/cohesity_job.py

```diff
@@ -778,6 +778,34 @@ def update_vmware_job(module, job_meta_data, job_details):
         )
 
 
+def delete_sources(module, job_meta_data, job_details):
+    job_details['environment'] = 'Physical'
+    for source in module.params.get('protection_sources'):
+        job_details['endpoint'] = source['endpoint']
+        source_id = get__prot_source_id__by_endpoint(
```

Comment:
Good stuff, we aren't telling users to supply ids, 

Can you put this in an try catch exception ?